docs/DeveloperGuide.md

```diff
@@ -320,7 +364,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-**Use case: UC02 - Delete a student**
+**Use case: UC04 - Find a student**
+
+**MSS**
+
+1.  User requests to find a student.
+2.  User provides the parameters to find student.
+3.  Tutor's Pet finds the student.
+4.  Student is found by Tutor's Pet.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The parameters provided are invalid.
+
+    * 2a1. Tutor's Pet shows an error message.
+
+      Use case ends.
+
+* 3a. Student not found.
+
+    * 3a1. Tutor's Pet shows that student is not found.
```

Comment:
Would be good to mention that it is a message, otherwise might be confusing how to show an absence of something.